custom-nb-image/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM  quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8-4c8f26e
+FROM  quay.io/opendatahub/notebooks:jupyter-minimal-ubi9-python-3.9-2023a_20230302
 # Install: torch (v1.12), ray (v2.1.0) and others
```

Comment:
nit:
can we update/  change this to leave out the versions? I think its pretty self explanatory anyway

Reply:
I'll wait until after that comment is removed/updated to approve, @KPostOffice